Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1352,7 +1352,8 @@ For more information, ensure that you have the fix `Pull Request <https://github
 Limitations
 ###########
 
-    * Cilium's eBPF kube-proxy replacement currently cannot be used with :ref:`gsg_encryption`.
+    * Cilium's eBPF kube-proxy replacement currently cannot be used with :ref:`encryption_ipsec`
+      (but works with :ref:`encryption_wg`).
```

Comment:
Please mention that the service traffic won't be encrypted with WG.

Reply:
I know you like to have a very very long KPR guide :sweat_smile: , but is it really the place for this? Remember we're in the limitations for KPR here, not the limitations for WireGuard. If we really can't mention WireGuard without stating its main limitation, I'd rather avoid to mention WireGuard at all.

Reply:
I think it is - otherwise users might mistakenly think that with WG their service traffic will be encrypted. I'd rather be verbose here.

Reply:
I think that's the main issue with this guide: you try to be verbose about everything. It's a bit as if you were writing a sentence and putting every single word in bold.

Reply:
The guide is indeed verbose, and it should be renamed to "Comprehensive guidebook to Cilium's LB". However, I still think that mentioning that the WG won't encrypt the traffic is important, as otherwise the sentence might easily mislead users.

Reply:
Maybe we should add these details to the WireGuard page instead, there we also have the space to explain that some KPR traffic like pod2clusterip will be encrypted, while nodeport2pod will not.